test/WebJobs.Script.Tests/Microsoft.Azure.WebJobs.Script.WebHost.deps.json

```diff
@@ -608,7 +609,7 @@
           }
         }
       },
-      "Microsoft.Azure.Functions.JavaWorker/2.1.0-SNAPSHOT": {},
+      "Microsoft.Azure.Functions.JavaWorker/2.0.0": {},
```

Comment:
Why the change here?

Reply:
Likely a result of the rebase/merge, but this doesn't have any effect on validation.